.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,52 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: The name of the Tarantool build artifact
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    strategy:
+      matrix:
+        metrics-version: [ "0.12.0" ]
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone the crud module
+        uses: actions/checkout@v3
+        with:
+          repository: slavakirichenko/crud
+          ref: 'slavakirichenko/add-reusable-testing-workflow'
+
+      - name: Download the Tarantool build artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ inputs.artifact_name }}
+
+      - name: Install Tarantool
+        # Now we're lucky: all dependencies are already installed. Check package
+        # dependencies when migrating to other OS version.
+        run: sudo dpkg -i tarantool_*.deb tarantool-common_*.deb tarantool-dev_*.deb
+
+      - name: Install requirements for community
+        run: |
+          tarantool --version
+          ./deps.sh
+
+      - run: echo $PWD/.rocks/bin >> $GITHUB_PATH
+
+      - name: Install metrics
+        run: tarantoolctl rocks install metrics ${{ matrix.metrics-version }}
+
+      # This server starts and listen on 8084 port that is used for tests
+      - name: Stop Mono server
+        run: sudo kill -9 $(sudo lsof -t -i tcp:8084) || true
+
+      - run: cmake -S . -B build
+
+      - name: Run tests and code coverage analysis
+        run: make -C build coverage
```

Comment:
We don't need coverage in an integration test. Is there a way to run tests without coverage?

Reply:
As far as I understand, there is no CMake target to run tests without coverage. But you may add one, it shouldn't be too complicated.

Reply:
Running tests without coverage is usually faster than with it. @slavakirichenko could you please add a new target?

Reply:
Try this:
```cmake
add_custom_target(test
  COMMAND ${LUATEST} -v
  WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
  COMMENT "Run regression tests without coverage"
)
```